pkg/controller/ttl/ttl_controller_test.go

```diff
@@ -235,7 +237,9 @@ func TestDesiredTTL(t *testing.T) {
 			boundaryStep:      testCase.boundaryStep,
 		}
 		if testCase.addNode {
-			ttlController.addNode(&v1.Node{})
+			_, ctx := ktesting.NewTestContext(t)
```

Comment:
```suggestion
			logger, _ := ktesting.NewTestContext(t)
```

Then remove the next line.